src/microbe_stage/MicrobeStage.cs

```diff
@@ -252,7 +252,7 @@ public void SetupStage()
         if (IsLoadedFromSave)
         {
             HUD.OnEnterStageTransition(false);
-            UpdatePatchSettings(true);
+            UpdatePatchSettings(true, Player.Transform.origin);
```

Comment:
Why did you change this to require the player position? The player position is already passed in on the spawn cycle, it shouldn't require setup here.

Reply:
When testing my changes, it felt weird when you change patches and the screen is empty. So I decided to quickly added some free cloud spawns on patch change. In hindsight, yeah this isn't a good way to implement that.

Reply:
I think it is a good change to make some stuff spawn on patch change. I *think* in the cloud setup, there's already a parameter or something that detects when the patch changes (as it should clear the old spawners). At least the code that calls the patch setup should know if a patch change is happening, and could pass that information to either here or the spawn system, which might be a bit nicer way to do it (basically reset the spawn intervals to cause something to spawn immediately).